modules/vaos/spec/request/messages_request_spec.rb

```diff
@@ -4,45 +4,145 @@
 RSpec.describe 'vaos appointment request messages', type: :request do
   include SchemaMatchers
 
-  let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
-
   before do
     Flipper.enable('va_online_scheduling')
     sign_in_as(current_user)
     allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
   end
 
-  context 'loa1 user with flipper enabled' do
-    let(:current_user) { build(:user, :loa1) }
+  describe 'GET messages' do
+    let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
+
+    context 'loa1 user with flipper enabled' do
+      let(:current_user) { build(:user, :loa1) }
 
-    it 'does not have access' do
-      get "/v0/vaos/appointment_requests/#{request_id}/messages"
-      expect(response).to have_http_status(:forbidden)
-      expect(JSON.parse(response.body)['errors'].first['detail'])
-        .to eq('You do not have access to online scheduling')
+      it 'does not have access' do
+        get "/v0/vaos/appointment_requests/#{request_id}/messages"
+        expect(response).to have_http_status(:forbidden)
+        expect(JSON.parse(response.body)['errors'].first['detail'])
+          .to eq('You do not have access to online scheduling')
+      end
+    end
+
+    context 'loa3 user' do
```

Comment:
nitpick but contexts read better imo when they start with "when" or "with"

Reply:
i'll make a note to change this in all our specs as part of a refactor PR I have planned.